doc/authoring_command_modules/authoring_commands.md

```diff
@@ -52,9 +52,11 @@ class MyCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.sdk.util import CliCommandType
+        from azure.cli.core.profiles._shared import MGMT_MYTYPE
         mymod_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.mymod.custom#{}')
+
         super(MyCommandsLoader, self).__init__(cli_ctx=cli_ctx,
-                                               min_profile='2017-03-10-profile',
+                                               resource_type=MGMT_MYTYPE,
```

Comment:
@limingu please confirm this is accurate.

Reply:
MGMT_MYTYPE needs to be added into azure.cli.core.profiles._shared.

Reply:
I added a blurb about that and a link to the more detailed "MultiAPI Modules" topic, which covers this in detail.